docs/reference/commandline/run.md

```diff
@@ -15,137 +15,106 @@ Aliases:
-                                      'container:<name|id>': reuse another container's network stack
-                                      'host': use the Docker host network stack
-                                      '<network-name>|<network-id>': connect to a user-defined network
-      --no-healthcheck                Disable any container-specified HEALTHCHECK
-      --oom-kill-disable              Disable OOM Killer
-      --oom-score-adj int             Tune host's OOM preferences (-1000 to 1000)
-      --pid string                    PID namespace to use
-      --pids-limit int                Tune container pids limit (set -1 for unlimited)
-      --privileged                    Give extended privileges to this container
-  -p, --publish value                 Publish a container's port(s) to the host (default [])
-  -P, --publish-all                   Publish all exposed ports to random ports
-      --pull string                   Pull image before running ("always"|"missing"|"never") (default "missing")
-  -q, --quiet                         Suppress the pull output
-      --read-only                     Mount the container's root filesystem as read only
-      --restart string                Restart policy to apply when a container exits (default "no")
-                                      Possible values are : no, on-failure[:max-retry], always, unless-stopped
-      --rm                            Automatically remove the container when it exits
-      --runtime string                Runtime to use for this container
-      --security-opt value            Security Options (default [])
-      --shm-size bytes                Size of /dev/shm
-                                      The format is `<number><unit>`. `number` must be greater than `0`.
-                                      Unit is optional and can be `b` (bytes), `k` (kilobytes), `m` (megabytes),
-                                      or `g` (gigabytes). If you omit the unit, the system uses bytes.
-      --sig-proxy                     Proxy received signals to the process (default true)
-      --stop-signal string            Signal to stop the container
-      --stop-timeout int              Timeout (in seconds) to stop a container
-      --storage-opt value             Storage driver options for the container (default [])
-      --sysctl value                  Sysctl options (default map[])
-      --tmpfs value                   Mount a tmpfs directory (default [])
-  -t, --tty                           Allocate a pseudo-TTY
-      --ulimit value                  Ulimit options (default [])
-  -u, --user string                   Username or UID (format: <name|uid>[:<group|gid>])
-      --userns string                 User namespace to use
-                                      'host': Use the Docker host user namespace
-                                      '': Use the Docker daemon user namespace specified by `--userns-remap` option.
```

Comment:
Here as well (wonder if we had these in the past or if they were always docs-only)